test/sync_to_ml.ml.expected

```diff
@@ -19,5 +19,6 @@ let f = "hello world!"
 
 [@@@part "42"] ;;
 
+
```

Comment:
a new extra newline ? :-)

Reply:
Nice catch. Changed the code to trim sooner.